prover/setup_gpu.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+# This script sets up the necessary data needed by the GPU FRI prover to be used locally.
+
+if [[ -z "${ZKSYNC_HOME}" ]]; then
+  echo "Environment variable ZKSYNC_HOME is not set. Make sure it's set and pointing to the root of this repository"
+  exit 1
+fi
+
+sed -i.backup 's/^proof_sending_mode=.*$/proof_sending_mode="OnlyRealProofs"/' ../etc/env/base/eth_sender.toml
```

Comment:
lots of stuff is common with setup.sh .. maybe we should have a common script (with a flag), or 2 files, but they call common utils?

Reply:
I added a flag to the script to be able to pick between cpu and gpu. https://github.com/matter-labs/zksync-era/pull/258/commits/3435d1b8ce22056cad473965b937e3e7f59a1c72 & https://github.com/matter-labs/zksync-era/pull/258/commits/54b2c0ebb11d6617a46de18017887b1079c41057